MIGRATING.md

```diff
@@ -2,6 +2,94 @@
+
+### Breaking Changes
+
+The following is a list of changes that break backward compatibility in version 3.0.
+
+**Okta.Sdk.OktaClient**
+- `CreatedScoped(Okta.Sdk.RequestContext requestContext)`
+<br />&mdash; Renamed `CreateScoped(Okta.Sdk.RequestContext requestContext)`
+
+**Okta.Sdk.GroupsClient**                
+- `ListGroups(String q,String filter,String after,Int32 limit,String expand)` 
+<br />&mdash; Signature changed `ListGroups(String q,String filter,String after,Int32 limit)`
+- `ListRules(Int32 limit,String after,String expand)` 
+<br />&mdash; Renamed with new signature `ListGroupRules(Int32 limit,String after,String search,String expand)`
+- `CreateRuleAsync(Okta.Sdk.IGroupRule groupRule,CancellationToken cancellationToken)` 
+<br />&mdash; Renamed `CreateGroupRuleAsync(Okta.Sdk.IGroupRule groupRule,CancellationToken cancellationToken)`
+- `DeleteRuleAsync(String ruleId,Boolean removeUsers,CancellationToken cancellationToken)` 
+<br />&mdash; Renamed with new signature `DeleteGroupRuleAsync(String ruleId,CancellationToken cancellationToken)`
+- `GetRuleAsync(String ruleId,String expand,CancellationToken cancellationToken)` 
+<br />&mdash; Renamed `GetGroupRuleAsync(String ruleId,String expand,CancellationToken cancellationToken)`
+- `UpdateRuleAsync(Okta.Sdk.IGroupRule groupRule,String ruleId,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `UpdateGroupRuleAsync(Okta.Sdk.IGroupRule groupRule,String ruleId,CancellationToken cancellationToken)`
+- `ActivateRuleAsync(String ruleId,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `ActivateGroupRuleAsync(String ruleId,CancellationToken cancellationToken)`
+- `DeactivateRuleAsync(String ruleId,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `DeactivateGroupRuleAsync(String ruleId,CancellationToken cancellationToken)`
+- `GetGroupAsync(String groupId,String expand,CancellationToken cancellationToken)`
+<br />&mdash; Signature changed `GetGroupAsync(String groupId,CancellationToken cancellationToken)`
+- `ListGroupUsers(String groupId,String after,Int32 limit,String managedBy)`
+<br />&mdash; Signature changed `ListGroupUsers(String groupId,String after,Int32 limit)`
+- `RemoveGroupUserAsync(String groupId,String userId,CancellationToken cancellationToken)`
+<br />&mdash; Renamed `RemoveUserFromGroupAsync(String groupId,String userId,CancellationToken cancellationToken)`
+
+**Okta.Sdk.PoliciesClient**
+- `ListPolicies(String type,String status,String after,Int32 limit,String expand)`
+<br />&mdash; Signature changed `ListPolicies(String type,String status,String expand)`
+- `AddPolicyRuleAsync(Okta.Sdk.IPolicyRule policyRule,String policyId,Boolean activate,CancellationToken cancellationToken)`
```

Comment:
Question: Should we replace `Boolean` by `bool`?

Reply:
Reflection; I'll update.